ci/run-couchbase-server-v5.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [ "$PUBLISH_SNAPSHOTS" == "false" ]; then
+    #while sleep 9m; do echo -e '\n=====[ Gradle build is still running ]====='; done &
+
+    echo "Pulling Couchbase docker image..."
+    docker pull couchbase/server:4.6.4
```

Comment:
@dima767 how did you validate against Couchbase 5? This is still pulling 4.6.4, no?